app/models/user.rb

```diff
@@ -128,6 +128,10 @@ def payload
     attrs
   end
 
+  def private_payload
+    payload.merge("mfa" => I18n.t("settings.edit.mfa.level.#{mfa_level}"))
```

Comment:
I would've expected to give the symbolic name here (`disabled` etc), because API clients will want a predictable value to match against. Or am I missing something obvious?

Reply:
Good call, updated in https://github.com/rubygems/rubygems.org/commit/f5d917686ad8d047d51158926b3cbcfd0aa88fdf